"Accepts 1 arg(s), received 198..." on every prompt with Zsh

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What happened?

After updating oh-my-posh to 26.4.1 i get a `Error: accepts 1 arg(s), received 198...` after every command i type in a zsh shell and it always has to do with `oh-my-posh print`

This only happens on zsh that i've tested. Tried it on PowerShell and it works normally.

![Image](https://github.com/user-attachments/assets/f97932cb-d976-4056-ab82-a6782010abcb)

### Theme

default theme/config

### What OS are you seeing the problem on?

Linux, Windows

### Which shell are you using?

zsh, powershell

### Log output

```Shell
Version: 26.4.1

Shell: zsh (5.9)

Prompt:

Arch ~/
>

Segments:

ConsoleTitle(true)                         -   0 ms
Python(false)                              -   1 ms
Path(true)                                 -   0 ms
Git(false)                                 -   0 ms
Executiontime(false)                       -   0 ms
Text(true)                                 -   0 ms

Run duration: 3.806751ms

Cache path: /home/dazuki/.cache/oh-my-posh

Config path: /home/dazuki/.config/ohmyposh/dazuki.toml

Logs:

[DEBUG] 01:57:37.318 debug.go:36 → debug mode enabled
[DEBUG] 01:57:37.318 load.go:Path:79 → config set using POSH_THEME: /home/dazuki/.config/ohmyposh/dazuki.toml
[TRACE] 01:57:37.318 load.go:Path() - 14.4µs
[TRACE] 01:57:37.318 load.go:loadConfig() - 111.259µs
[TRACE] 01:57:37.318 load.go:Load() - 116.769µs
[DEBUG] 01:57:37.318 terminal.go:Init:54 → plain mode enabled
[DEBUG] 01:57:37.318 home.go:12 → /home/dazuki
[TRACE] 01:57:37.318 path.go:Path() - 144.738µs
[DEBUG] 01:57:37.318 file.go:Init:26 → loading cache file: /home/dazuki/.cache/oh-my-posh/omp.cache
[DEBUG] 01:57:37.318 file.go:Init:47 → loading cache key: INITVERSIONZSH
[DEBUG] 01:57:37.318 file.go:Init:47 → loading cache key: environment_platform
[DEBUG] 01:57:37.318 file.go:Init:47 → loading cache key: is_wsl
[DEBUG] 01:57:37.318 file.go:Init:47 → loading cache key: upgrade
[DEBUG] 01:57:37.318 file.go:Init:47 → loading cache key: upgrade_check
[TRACE] 01:57:37.318 file.go:Init(/home/dazuki/.cache/oh-my-posh/omp.cache) - 57.909µs
[DEBUG] 01:57:37.318 home.go:12 → /home/dazuki
[TRACE] 01:57:37.318 path.go:Path() - 9.73µs
[DEBUG] 01:57:37.318 file.go:Init:26 → loading cache file: /home/dazuki/.cache/oh-my-posh/omp.cache.74f06bff-a712-4422-8c13-c69d2d267d82
[DEBUG] 01:57:37.319 file.go:Init:47 → loading cache key: prompt_count_cache
[DEBUG] 01:57:37.319 file.go:Init:47 → loading cache key: template_cache
[TRACE] 01:57:37.319 file.go:Init(/home/dazuki/.cache/oh-my-posh/omp.cache.74f06bff-a712-4422-8c13-c69d2d267d82) - 29.68µs
[DEBUG] 01:57:37.319 file.go:Get:80 → getting cache key: prompt_count_cache with value: 18
[TRACE] 01:57:37.319 terminal.go:setPromptCount() - 2.64µs
[TRACE] 01:57:37.319 terminal.go:GOOS() - 50ns
[DEBUG] 01:57:37.319 terminal.go:setPwd:118 → /home/dazuki
[TRACE] 01:57:37.319 terminal.go:setPwd() - 8.909µs
[TRACE] 01:57:37.319 terminal.go:Init() - 412.303µs
[TRACE] 01:57:37.319 terminal.go:Shell() - 60ns
[TRACE] 01:57:37.319 terminal.go:Flags() - 50ns
[DEBUG] 01:57:37.319 file.go:Get:80 → getting cache key: template_cache with value: {"SegmentsCache":{"Path":{"Segment":{"Text":"Arch \u003cgreen\u003e~/\u003c/\u003e","Index":0},"Location":"//wsl.localhost/Arch/home/dazuki","Path":"~/","Folders":[],"StackCount":0,"Writable":true,"RootDir":false},"Text":{"Segment":{"Text":"\u003e","Index":0}}},"Segments":{},"Var":{},"PWD":"~","Folder":"dazuki","PSWD":"","UserName":"dazuki","HostName":"arch-wsl","ShellVersion":"5.9","Shell":"zsh","AbsolutePWD":"//wsl.localhost/Arch/home/dazuki","OS":"arch","Version":"26.4.1","PromptCount":18,"SHLVL":1,"Jobs":0,"Code":1,"WSL":true,"Root":false}
[TRACE] 01:57:37.319 cache.go:restoreCache() - 72.119µs
[DEBUG] 01:57:37.319 writer.go:Init:106 → terminal program: Windows Terminal
[DEBUG] 01:57:37.319 writer.go:Init:107 → terminal shell: shell
[DEBUG] 01:57:37.319 terminal.go:Getenv:80 → NO DATA
[TRACE] 01:57:37.319 terminal.go:Getenv(OMP_CACHE_DISABLED) - 2.52µs
[DEBUG] 01:57:37.319 file.go:Get:75 → cache key not found: accent_color
[TRACE] 01:57:37.319 colors.go:SetAccentColor() - 5.19µs
[TRACE] 01:57:37.319 terminal.go:Shell() - 40ns
[DEBUG] 01:57:37.319 terminal.go:Getenv:80 → 5.9
[TRACE] 01:57:37.319 terminal.go:Getenv(POSH_SHELL_VERSION) - 44.86µs
[DEBUG] 01:57:37.319 debug.go:PrintDebug:24 → segment: Title
[DEBUG] 01:57:37.319 text.go:patchTemplate:125 → {{.HostName | upper}} :: {{ .PWD }}
[TRACE] 01:57:37.319 text.go:Render({{.HostName | upper}} :: {{ .PWD }}) - 178.278µs
[TRACE] 01:57:37.319 terminal.go:Flags() - 40ns
[DEBUG] 01:57:37.319 terminal.go:Getenv:80 → NO DATA
[TRACE] 01:57:37.319 terminal.go:Getenv(POSH_CURSOR_LINE) - 2.28µs
[DEBUG] 01:57:37.319 terminal.go:Getenv:80 → NO DATA
[TRACE] 01:57:37.319 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 1.31µs
[TRACE] 01:57:37.319 terminal.go:Flags() - 20ns
[TRACE] 01:57:37.319 terminal.go:Flags() - 20ns
[TRACE] 01:57:37.319 terminal.go:Flags() - 30ns
[TRACE] 01:57:37.319 terminal.go:Flags() - 210ns
[DEBUG] 01:57:37.319 home.go:12 → /home/dazuki
[TRACE] 01:57:37.319 terminal.go:GOOS() - 60ns
[DEBUG] 01:57:37.319 segment.go:Execute:112 → segment: Git
[DEBUG] 01:57:37.319 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 01:57:37.319 segment.go:isToggled:229 → no toggles found
[DEBUG] 01:57:37.319 home.go:12 → /home/dazuki
[TRACE] 01:57:37.319 terminal.go:Flags() - 100ns
[TRACE] 01:57:37.319 terminal.go:GOOS() - 90ns
[DEBUG] 01:57:37.319 file.go:Get:80 → getting cache key: is_wsl with value: true
[DEBUG] 01:57:37.319 segment.go:Execute:112 → segment: Python
[DEBUG] 01:57:37.319 terminal_unix.go:IsWsl:32 → true
[TRACE] 01:57:37.319 terminal_unix.go:IsWsl() - 7.56µs
[DEBUG] 01:57:37.319 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 01:57:37.319 segment.go:isToggled:229 → no toggles found
[DEBUG] 01:57:37.319 home.go:12 → /home/dazuki
[DEBUG] 01:57:37.319 terminal.go:FileContent:228 ↓
    6.15.1-WSL2-STABLE+

[TRACE] 01:57:37.319 terminal.go:FileContent(/proc/sys/kernel/osrelease) - 27.819µs
[DEBUG] 01:57:37.319 segment.go:Execute:112 → segment: Path
[TRACE] 01:57:37.319 terminal_unix.go:IsWsl2() - 43.439µs
[DEBUG] 01:57:37.319 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 01:57:37.319 segment.go:isToggled:229 → no toggles found
[DEBUG] 01:57:37.319 properties.go:GetStringArray:52 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 01:57:37.319 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 01:57:37.319 properties.go:GetBool:22 → display_cygpath: false
[TRACE] 01:57:37.319 terminal.go:GOOS() - 60ns
[TRACE] 01:57:37.319 separator.go:Separator() - 60ns
[TRACE] 01:57:37.319 terminal.go:Shell() - 150ns
[TRACE] 01:57:37.319 terminal.go:Shell() - 40ns
[TRACE] 01:57:37.319 terminal.go:GOOS() - 40ns
[DEBUG] 01:57:37.319 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 01:57:37.319 properties.go:GetString:28 → windows_registry_icon: 
[DEBUG] 01:57:37.319 properties.go:GetString:28 → windows_registry_icon: 
[DEBUG] 01:57:37.319 home.go:12 → /home/dazuki
[TRACE] 01:57:37.319 separator.go:Separator() - 50ns
[TRACE] 01:57:37.319 terminal.go:GOOS() - 70ns
[TRACE] 01:57:37.319 terminal.go:GOOS() - 70ns
[DEBUG] 01:57:37.319 properties.go:GetString:28 → home_icon: ~/
[DEBUG] 01:57:37.319 properties.go:GetKeyValueMap:46 → mapped_locations: map[/c/:<magenta>C:</>]
[TRACE] 01:57:37.319 text.go:Render(/c/) - 110ns
[TRACE] 01:57:37.319 separator.go:Separator() - 60ns
[TRACE] 01:57:37.319 terminal.go:GOOS() - 50ns
[TRACE] 01:57:37.319 terminal.go:GOOS() - 50ns
[DEBUG] 01:57:37.319 home.go:12 → /home/dazuki
[TRACE] 01:57:37.320 separator.go:Separator() - 60ns
[TRACE] 01:57:37.320 terminal.go:GOOS() - 40ns
[TRACE] 01:57:37.320 terminal.go:GOOS() - 50ns
[TRACE] 01:57:37.320 separator.go:Separator() - 50ns
[TRACE] 01:57:37.320 terminal.go:GOOS() - 30ns
[TRACE] 01:57:37.320 terminal.go:GOOS() - 40ns
[TRACE] 01:57:37.320 terminal.go:GOOS() - 40ns
[TRACE] 01:57:37.320 terminal.go:GOOS() - 50ns
[TRACE] 01:57:37.320 terminal.go:GOOS() - 40ns
[TRACE] 01:57:37.320 terminal.go:Flags() - 60ns
[DEBUG] 01:57:37.320 properties.go:GetStringArray:52 → cycle: []
[DEBUG] 01:57:37.320 properties.go:GetString:28 → folder_separator_template:
[DEBUG] 01:57:37.320 properties.go:GetString:28 → folder_separator_icon: /
[DEBUG] 01:57:37.320 properties.go:GetBool:22 → cycle_folder_separator: false
[DEBUG] 01:57:37.320 properties.go:GetString:28 → folder_format: %s
[DEBUG] 01:57:37.320 properties.go:GetString:28 → edge_format: %s
[DEBUG] 01:57:37.320 properties.go:GetString:28 → left_format: %s
[DEBUG] 01:57:37.320 properties.go:GetString:28 → right_format: %s
[TRACE] 01:57:37.320 terminal.go:Flags() - 40ns
[TRACE] 01:57:37.320 terminal.go:GOOS() - 30ns
[TRACE] 01:57:37.320 terminal.go:StackCount() - 60ns
[TRACE] 01:57:37.320 terminal_unix.go:DirIsWritable(/home/dazuki) - 4.55µs
[DEBUG] 01:57:37.320 properties.go:GetBool:22 → fetch_virtual_env: true
[DEBUG] 01:57:37.320 terminal.go:CommandPath:320 → /usr/sbin/python
[TRACE] 01:57:37.320 terminal.go:CommandPath(python) - 68.909µs
[DEBUG] 01:57:37.320 terminal.go:RunCommand:296 → //wsl.localhost/Arch/home/dazuki
[TRACE] 01:57:37.320 terminal.go:RunCommand(wslpath -m /home/dazuki) - 750.609µs
[TRACE] 01:57:37.320 terminal.go:GOOS() - 50ns
[DEBUG] 01:57:37.320 terminal.go:CommandPath:320 → /usr/sbin/git
[TRACE] 01:57:37.320 terminal.go:CommandPath(git) - 31.769µs
[TRACE] 01:57:37.320 terminal.go:HasCommand(git) - 33.749µs
[DEBUG] 01:57:37.320 properties.go:GetBool:22 → fetch_bare_info: false
[DEBUG] 01:57:37.320 terminal.go:ResolveSymlink:211 → /home/dazuki
[TRACE] 01:57:37.320 terminal.go:ResolveSymlink(/home/dazuki) - 5.86µs
[ERROR] 01:57:37.320 terminal.go:HasParentFilePath:479 → stat .git: no such file or directory
[TRACE] 01:57:37.320 terminal.go:HasParentFilePath(.git) - 22.27µs
[DEBUG] 01:57:37.321 terminal.go:HasFilesInDir:167 → false
[TRACE] 01:57:37.321 terminal.go:HasFilesInDir(pyvenv.cfg) - 1.098454ms
[DEBUG] 01:57:37.321 terminal.go:HasFilesInDir:167 → false
[TRACE] 01:57:37.321 terminal.go:HasFilesInDir(pyvenv.cfg) - 33.919µs
[DEBUG] 01:57:37.321 properties.go:GetBool:22 → folder_name_fallback: true
[DEBUG] 01:57:37.321 properties.go:GetStringArray:52 → default_venv_names: [.venv venv]
[DEBUG] 01:57:37.321 terminal.go:Getenv:80 → NO DATA
[TRACE] 01:57:37.321 terminal.go:Getenv(VIRTUAL_ENV) - 2.97µs
[DEBUG] 01:57:37.321 terminal.go:Getenv:80 → NO DATA
[TRACE] 01:57:37.321 terminal.go:Getenv(CONDA_ENV_PATH) - 2.29µs
[DEBUG] 01:57:37.321 terminal.go:Getenv:80 → NO DATA
[TRACE] 01:57:37.321 terminal.go:Getenv(CONDA_DEFAULT_ENV) - 2.4µs
[TRACE] 01:57:37.321 text.go:Render(plain) - 240ns
[DEBUG] 01:57:37.321 text.go:patchTemplate:125 → Arch <green>{{ .Data.Path }}</>{{ if not .Data.Writable }} <red></>{{ end }}
[TRACE] 01:57:37.321 text.go:Render(Arch <green>{{ .Path }}</>{{ if not .Writable }} <red></>{{ end }}) - 96.469µs
[TRACE] 01:57:37.321 text.go:Render(plain) - 60ns
[TRACE] 01:57:37.321 text.go:Render(plain) - 70ns
[TRACE] 01:57:37.321 terminal.go:Shell() - 60ns
[TRACE] 01:57:37.321 terminal.go:Shell() - 20ns
[TRACE] 01:57:37.321 terminal.go:Flags() - 40ns
[DEBUG] 01:57:37.321 home.go:12 → /home/dazuki
[TRACE] 01:57:37.321 terminal.go:GOOS() - 40ns
[DEBUG] 01:57:37.321 segment.go:Execute:112 → segment: Executiontime
[DEBUG] 01:57:37.321 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 01:57:37.321 segment.go:isToggled:229 → no toggles found
[DEBUG] 01:57:37.321 properties.go:GetBool:22 → always_enabled: false
[TRACE] 01:57:37.321 terminal.go:ExecutionTime() - 70ns
[DEBUG] 01:57:37.321 properties.go:GetFloat64:34 → threshold: 5000.000000
[TRACE] 01:57:37.321 text.go:Render(plain) - 140ns
[TRACE] 01:57:37.321 terminal.go:Flags() - 40ns
[DEBUG] 01:57:37.321 home.go:12 → /home/dazuki
[TRACE] 01:57:37.321 terminal.go:GOOS() - 40ns
[DEBUG] 01:57:37.321 segment.go:Execute:112 → segment: Text
[DEBUG] 01:57:37.321 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 01:57:37.321 segment.go:isToggled:229 → no toggles found
[TRACE] 01:57:37.321 text.go:Render(>) - 60ns
[DEBUG] 01:57:37.321 text.go:patchTemplate:125 → {{if gt .Code 0}}red{{end}}
[TRACE] 01:57:37.321 text.go:Render({{if gt .Code 0}}red{{end}}) - 34.189µs
[TRACE] 01:57:37.321 text.go:Render(plain) - 90ns
[DEBUG] 01:57:37.321 text.go:patchTemplate:125 → {{if gt .Code 0}}red{{end}}
[TRACE] 01:57:37.321 text.go:Render({{if gt .Code 0}}red{{end}}) - 11.33µs
[TRACE] 01:57:37.321 terminal.go:Shell() - 40ns
[TRACE] 01:57:37.321 terminal.go:Shell() - 20ns
[TRACE] 01:57:37.321 terminal.go:Flags() - 40ns
[TRACE] 01:57:37.321 terminal.go:Shell() - 30ns
[TRACE] 01:57:37.321 terminal.go:Flags() - 30ns
[DEBUG] 01:57:37.321 home.go:12 → /home/dazuki
[TRACE] 01:57:37.321 path.go:Path() - 14.569µs
[TRACE] 01:57:37.321 terminal.go:Flags() - 30ns
```

--- a/src/shell/scripts/omp.zsh
+++ b/src/shell/scripts/omp.zsh
@@ -168,11 +168,11 @@ function _omp_zle-line-init() {
 
   # We need this workaround because when the `filler` is set,
   # there will be a redundant blank line below the transient prompt if the input is empty.
+  local terminal_width_option
   if [[ -z $BUFFER ]]; then
-    local options="--terminal-width=$((${COLUMNS-0} - 1))"
+    terminal_width_option="--terminal-width=$((${COLUMNS-0} - 1))"
   fi
-
-  eval "$(_omp_get_prompt transient --eval $options)"
+  eval "$(_omp_get_prompt transient --eval $terminal_width_option)"
   zle .reset-prompt
 
   if ((ret)); then

